### DISCUSSION TITLE
Support of Dropshopping API ?

### DISCUSSION THREAD

**Discussion by espinosa** [solidusio/solidus#4273] [New Features or Ideas]
Hi,
Does solidus support dropshopping providers like Printful or other ?
I have done some search but doesn't have found anything. I have found the solidus_multi_vendor gem but it's not updated since 2020 and not sure that can solve my use case.
Printful already include stores like Prestatshop, Shopify, Magento, etc.., and has also an API integration.
You can see more details here: https://www.printful.com/integrations
Thanks for you response.

**Comment by jarednorman** [comment]
There exist Solidus stores out there that use Printful, but I don't believe any of them have open-sourced their API integrations.

**Reply by espinosa** [reply]
Thanks for your response @jarednorman
